lib/rails_pdf/renderer.rb

```diff
@@ -13,23 +13,25 @@ def layout(layout)
     end
 
     def render(&block)
-      content = ApplicationController.render(file: @file, layout: @layout)
+      controller = ActionController::Base.new

```

Comment:
does it work with both rails 5 and 4?

if not we need some conditional rendering and check rails version

Reply:
Yes, it works, checked in ruby 2.6.1 and rails 5.2.3